usaspending_api/references/urls_autocomplete.py

```diff
@@ -1,11 +1,14 @@
 from django.conf.urls import url
 from usaspending_api.references.views_v2.autocomplete import AwardingAgencyAutocompleteViewSet, \
-    BudgetFunctionAutocompleteViewSet, FundingAgencyAutocompleteViewSet, RecipientAutocompleteViewSet
+    BudgetFunctionAutocompleteViewSet, FundingAgencyAutocompleteViewSet, \
+    NAICSAutocompleteViewSet, RecipientAutocompleteViewSet
 
 
 urlpatterns = [
     url(r'^awarding_agency', AwardingAgencyAutocompleteViewSet.as_view()),
     url(r'^budget_function', BudgetFunctionAutocompleteViewSet.as_view()),
     url(r'^funding_agency', FundingAgencyAutocompleteViewSet.as_view()),
-    url(r'^recipient/', RecipientAutocompleteViewSet.as_view())
+    url(r'^naics', NAICSAutocompleteViewSet.as_view()),
+    url(r'^recipient/', RecipientAutocompleteViewSet.as_view()),
```

Comment:
does the extra comma here mess with anything? probably should remove it anyways to denote the end of the list